foundations/javascript_basics/project_calculator.md

```diff
@@ -46,5 +46,5 @@ Here are some use cases (abilities your project needs to have):
 *   Users can get floating point numbers if they do the math required to get one, but they can't type them in yet. Add a `.` button and let users input decimals! Make sure you don't let them type more than one though: `12.3.56.5`. It is hard to do math on these numbers. \(disable the decimal button if there's already one in the display\)
 *   Make it look nice! This is a great project to practice your CSS skills. At least make the operations a different color from the keypad buttons.
 *   Add a "backspace" button, so the user can undo if they click the wrong number.
-*   Add keyboard support!
+*   Add keyboard support! You might run into an issue where keys such as (`/`) might cause you some trouble. Read up on `event.preventDefault();` [here](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) to help solve this issue.
```

Comment:
Can you edit your link text to be more accessible? A great explanation of why "here" isn't great is [available from WebAim](https://webaim.org/techniques/hypertext/#screen_readers)

Reply:
Done